client/state/reader/onboarding/handlers.ts

```diff
@@ -13,7 +13,7 @@ import { isProfileComplete } from './utils';
 
 function dispatchNotice( dispatch: Store[ 'dispatch' ] ) {
 	dispatch(
-		successNotice( translate( 'Profile complete!' ), {
+		successNotice( translate( 'Settings saved!' ), {
```

Comment:
I changed the wording here to reflect that your profile may not be 100% complete.